<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevoObispo</class>
 <widget class="QDialog" name="dlgNuevoObispo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Persona</string>
       </property>
       <property name="buddy">
        <cstring>txtPersona</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtPersona"/>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarPersona">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Di&amp;ócesis</string>
       </property>
       <property name="buddy">
        <cstring>txtDiocesis</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtDiocesis">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btQuitarDiocesis">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Papa</string>
       </property>
       <property name="buddy">
        <cstring>txtPapa</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPapa"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fecha inico</string>
       </property>
       <property name="buddy">
        <cstring>dtFechaInicio</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtFechaInicio">
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fecha fina&amp;l</string>
       </property>
       <property name="buddy">
        <cstring>dtFechaFinal</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtFechaFinal">
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Duración</string>
       </property>
       <property name="buddy">
        <cstring>txtDuracion</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDuracion"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="ckFechaAprox">
       <property name="text">
        <string>Fecha aprox</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckFinPorMuerte">
       <property name="text">
        <string>Fin por muerte</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckVolverMirar">
       <property name="text">
        <string>Volver a mirar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
